vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridHeaderFooterRowIT.java

```diff
@@ -257,6 +259,15 @@ public void addHeaderRow_setMultiselect_disableSelection() {
                 "0", headerCells.get(0).getText());
     }
 
+    @Test
+    public void multiSelectionToSingleselect() {
+        clickButton("set-singleselect");
+        Assert.assertEquals(findElements(By.tagName("vaadin-checkbox")).stream().count(), 0);
+        WebElement secondRow = findElements(By.tagName("vaadin-grid-cell-content")).get(4);
+        secondRow.click();
+        Assert.assertTrue(String.valueOf(secondRow.isSelected()),true);
```

Comment:
`Assert.assertTrue(...,true)` is pointless, it will always pass.

Reply:
done